mmdet3d/core/bbox/structures/utils.py

```diff
@@ -111,12 +111,14 @@ def get_box_type(box_type):
     return box_type_3d, box_mode_3d
 
 
-def points_cam2img(points_3d, proj_mat):
+def points_cam2img(points_3d, proj_mat, return_z=False):
```

Comment:
How about keeping the same with https://github.com/open-mmlab/mmdetection3d/blob/a6ee9335682967ad2d697f80cb79991028bb966b/mmdet3d/core/bbox/box_np_ops.py#L338